scripts/lib/CIME/XML/generic_xml.py

```diff
@@ -91,7 +187,7 @@ def get_node(self, nodename, attributes=None, root=None, xpath=None):
 
         nodes = self.get_nodes(nodename, attributes=attributes, root=root, xpath=xpath)
 
-        expect(len(nodes) == 1, "Incorrect number of matches, {:d}, for nodename '{}' and attrs '{}' in file '{}'".format(len(nodes), nodename, attributes, self.filename))
+        #expect(len(nodes) == 1, "Incorrect number of matches, {:d}, for nodename '{}' and attrs '{}' in file '{}'".format(len(nodes), nodename, attributes, self.filename))
```

Comment:
why remove these checks? Should they be enabled only for debug?   Should they be removed entirely?

Reply:
If you look at the profile, a significant chunk of computation is taken by these expects since we call get_nodes so often. I was just trying to see how much speed-up I could achieve. They add some value, so would probably have to be left in.